src/content/docs/pt-br/community-resources/content.mdx

```diff
@@ -0,0 +1,85 @@
+
+:::note[Nota dos tradutores]
+Apesar das descrições abaixo estarem traduzidas, o conteúdo dos links é em Inglês.
+:::
+
+## Cursos e Tutoriais
+
+Cursos e tutoriais são focados em ensinar você novos conceitos, geralmente com exercícios ou projetos de exemplo para construir. São ótimas maneiras de aprender novos conceitos e fornecer as ferramentas necessárias para trabalhar com Astro em seu projeto.
+
+Confira os seguintes cursos e tutoriais para aprender mais sobre Astro.
+
+### Tutoriais Introdutórios
+  - [Curso intensivo sobre o framework web Astro](https://www.youtube.com/watch?v=e-hTm5VmofI) por freeCodeCamp
+  - [Curso intensivo sobre Astro em 20 minutos](https://www.youtube.com/watch?v=zrPVTf761OI) por Chris Pennington (curso completo pago: [Aprenda Astro](https://learnastro.dev/))
+  - [Curso intensivo sobre Astro 3.0](https://www.youtube.com/watch?v=qBOz6TpYAOg) por James Q Quick (curso completo pago: [Construa Sites modernos com Astro](https://astrocourse.dev/))
+  - [Curso intenssivo sobre Astro em 60 minutos](https://www.youtube.com/watch?v=NniT0vKyn-E) por @developedbyed
+  - [Entendendo Astro (ebook)](https://www.ohansemmanuel.com/books/understanding-astro) por Ohans Emmanuel
+  - [O Vale do Código - Astro](https://thevalleyofcode.com/astro) por Flavio Copes
+
+  ### Vídeo Tutoriais
+  - [Curso rápido sobre Astro: Crie um blog com SSR](https://www.youtube.com/watch?v=XoIHKO6AkoM)
+  - [Como eu criei uma aplicação de filmes com Astro 3.x](https://www.youtube.com/watch?v=Gvr4WhgfP0w)
+  - [Você pode AINDA não ter entendido coleções de conteúdo…](https://www.youtube.com/watch?v=Fcw4c3wzm7I)
+  - [Construa uma plataforma customizada de blog com Astro e Appwrite](https://www.youtube.com/watch?v=OERqwLy_reA)
+  - [Curso intensivo sobre Astro - Criando um portfólio](https://www.youtube.com/watch?v=TwWvNK0yHjI)
+  - [Criando um blog full stack com Astro](https://egghead.io/courses/build-a-full-stack-blog-with-astro-7ffcf9ec)
+
+## Receitas e Guias
+
+Veja exemplos guiados de como adicionar funcionalidades no seu projeto Astro, enviados por pessoas da nossa comunidade! 
+
+[Receitas](/pt-br/recipes/) são tipicamente curtas, focadas em guias passo a passo que te conduzem por meio da conclusão de um exemplo prático de uma tarefa específica. Receitas são uma excelente maneira de adicionar funcionalidades ou comportamentos ao seu projeto Astro seguindo instruções passo a passo! 
+
+Outros guias podem explicar conceitos relacionados a uma área de conteúdo, como por exemplo usar imagens ou trabalhar com MDX.
+
+:::tip[Adicione a sua!]
+Escreveu uma receita para o Astro? [Edite essa página](https://github.com/withastro/docs/edit/main/src/content/docs/en/community-resources/content.mdx) e adicione seu link na seção apropriada!
+:::
+
```

Comment:
I kept most of the translations from the old page 
[PR reference](https://github.com/withastro/docs/pull/6775/files)